moveit_ros/moveit_servo/config/test_config_panda.yaml

```diff
@@ -29,6 +30,7 @@ publish_joint_accelerations: false
 ## Plugins for smoothing outgoing commands
 use_smoothing: false
 smoothing_filter_plugin_name: "online_signal_smoothing::ButterworthFilterPlugin"
+joint_acceleration_limit: 500.0
```

Comment:
again, remove this line in favor of `joint_acceleration` up top.

Reply:
Done